modules/core/src/test/java/org/apache/ignite/internal/processors/cache/query/continuous/GridCacheContinuousQueryAbstractSelfTest.java

```diff
@@ -105,6 +111,12 @@
     /** */
     private static final String NO_CACHE_IGNITE_INSTANCE_NAME = "noCacheGrid";
 
+    /**
+     * Remote filter latch. Executed on remote nodes, so must be transient to not get serialization exception.
```

Comment:
Remote filter latch is executed on remote nodes, so must be transient to **do** not get serialization exception.